packages/admin-app/src/components/GoHomeButton/GoHomeButton.tsx

```diff
@@ -0,0 +1,42 @@
+import HomeIcon from '@material-ui/icons/Home';
+import { ButtonProps, makeStyles, Link } from '@material-ui/core';
+import React from 'react';
+import { useIntl } from 'react-intl';
+import { GenericMessage } from '../../lang';
+
+export interface GoHomeButtonProps extends ButtonProps {
+    id: string;
+    defaultMessage?: string;
+    description?: string;
+}
+
+const useStyles = makeStyles(() => ({
+    goHome: {
+        textTransform: 'uppercase',
+        backgroundColor: '#1A237E',
+        color: 'white',
+        padding: 10,
+        borderRadius: 5,
+        fontSize: 14,
+    },
+    icon: {
+        top: 5,
+        position: 'relative',
+        marginRight: 5,
+    },
+}));
+
+export const GoHomeButton: React.FC<GoHomeButtonProps> = (props) => {
```

Comment:
Even though this go home button is only currently used once, it has so much styling it felt like it should be its own component.  Also, I may grow it into something more reusable as I work on subsequent pages.